<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="openFilesGedit">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Открыть файлы
 в текстовом редакторе</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="createFile">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Создать файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="uncrypt">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Расшифровать файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="zip">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Архивировать файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="unzip">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Разархивировать файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="crypt">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Зашифровать файлы</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="removeFile">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Удалить файлы</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="fileNames">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
